apps/docs/pages/guides/database/extensions/pgaudit.mdx

```diff
@@ -59,6 +59,14 @@ The `pgaudit.log` setting controls which statements to log. Available values inc
 
 For a full list of available settings see [settings docs](https://github.com/pgaudit/pgaudit/blob/master/README.md#settings). Be aware that the `all` setting will generate a very large volume of logs.
 
+<Admonition>
+
+We don't currently support configuring `pgaudit.log_parameter` because it may log secrets in encrypted columns if you happen to be using [pgsodium](/docs/guides/database/extensions/pgsodium) or [Vault](/docs/guides/database/vault).
```

Comment:
```suggestion
We don't currently support configuring `pgaudit.log_parameter` because it may log secrets in encrypted columns if you are using [pgsodium](/docs/guides/database/extensions/pgsodium) or [Vault](/docs/guides/database/vault).
```